### PR TITLE
[Encode] Add some missing device IDs.

### PR DESCRIPTION
**Pull request by Tianhaol**
Fixes: #507

### CODE REVIEW

File: media_driver/linux/common/os/i915/include/intel_chipset.h [hunk 1]
```diff
@@ -220,15 +220,21 @@
 
 #define PCI_CHIP_COFFEELAKE_S_GT1_1     0x3E90
 #define PCI_CHIP_COFFEELAKE_S_GT1_2     0x3E93
+#define PCI_CHIP_COFFEELAKE_S_GT1_3     0x3E99
 #define PCI_CHIP_COFFEELAKE_S_GT2_1     0x3E91
 #define PCI_CHIP_COFFEELAKE_S_GT2_2     0x3E92
 #define PCI_CHIP_COFFEELAKE_S_GT2_3     0x3E96
 #define PCI_CHIP_COFFEELAKE_H_GT2_1     0x3E9B
+#define PCI_CHIP_COFFEELAKE_H_GT1_1     0x3E9C
```
Comment (dvrogozh) [inline media_driver/linux/common/os/i915/include/intel_chipset.h]: This device ID is not yet known to the kernel. Wait till https://patchwork.freedesktop.org/patch/282347/ will land. Please, provide a separate PR.

File: media_driver/linux/common/os/i915/include/intel_chipset.h [hunk 2]
```diff
@@ -220,15 +220,21 @@
 
 #define PCI_CHIP_COFFEELAKE_S_GT1_1     0x3E90
 #define PCI_CHIP_COFFEELAKE_S_GT1_2     0x3E93
+#define PCI_CHIP_COFFEELAKE_S_GT1_3     0x3E99
```
Comment (dvrogozh) [inline media_driver/linux/common/os/i915/include/intel_chipset.h]: For this device Id reference will be:
`c99d783 ("drm/i915/cfl: Adding more Coffee Lake PCI IDs.")`

Please, check that you add other device ids from the above kernel commit, i.e.: 0x3e99, 0x3E9A (you miss it), 0x3EA0, 0x3EA1, 0x3EA2, 0x3EA3, 0x3EA4, 0x3EA9

Comment (Tianhaol) [reply media_driver/linux/common/os/i915/include/intel_chipset.h]: Added these device IDs.

File: media_driver/linux/common/os/i915/include/intel_chipset.h (hunk 1 above)
Comment (Tianhaol) [reply media_driver/linux/common/os/i915/include/intel_chipset.h]: Since this patch is submitted, keep this device ID.

Comment (Tianhaol) [reply media_driver/linux/common/os/i915/include/intel_chipset.h]: Added the device ID needed by #507

**Pull request merged**